src/server/models/relationship.model.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+import mongoose from 'mongoose';
+
+const Schema = mongoose.Schema;
+let relationshipSchema = new Schema({
+    mentorId: String,
+    menteeId: String,
+    check: Boolean,
```

Comment:
instead of "check", we should call it "status", and make that an enum to store bits like PENDING, COMPLETE etc. 
Or at the very least, we can call this matchComplete: Boolean.